sdk/quantum/quantum-jobs/package.json

```
@@ -1,116 +1,124 @@
 {
   "name": "@azure/quantum-jobs",
-  "author": "Microsoft Corporation",
-  "description": "Azure Quantum REST API client",
   "version": "1.0.0-beta.1",
-  "dependencies": {
-    "@azure/core-paging": "^1.1.1",
-    "@azure/core-http": "^1.2.0",
-    "tslib": "^2.0.0",
-    "@opentelemetry/api": "^0.10.2"
+  "description": "An isomorphic client library for Azure Quantum",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "types": "types/index.d.ts",
+  "engines": {
+    "node": ">=8.0.0"
   },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1 && npm run extract-api",
+    "build": "npm run build:node && npm run build:browser",
+    "build:test:browser": "",
+    "build:test:browser1": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test": "npm run build:test:node && npm run build:test:browser",
+    "build:samples": "echo Skipped.",
+    "check-format": "prettier --list-different --config ../../.prettierrc.json --ignore-path ../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm dist-browser test-dist test-browser types *.tgz *.log",
+    "coverage": "nyc --reporter=lcov --exclude-after-remap=false mocha -t 120000 dist-test/index.node.js --reporter ../../../common/tools/mocha-multi-reporter.js",
+    "execute:js-samples": "node ../../../common/scripts/run-samples.js samples/javascript/",
+    "execute:samples": "npm run build:samples && dev-tool samples run dist-samples/javascript dist-samples/typescript/dist/dist-samples/typescript/src/",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../.prettierrc.json --ignore-path ../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "npm run build:test:browser && cross-env TEST_MODE=live karma start --single-run",
```

```suggestion
    "integration-test:browser": "karma start --single-run",
```
This command is used in the live test pipelines, `build:test:browser` would be duplication, and we recommend not having TEST_MODE variable in package.json scripts since that would restrict running the test in other modes..

---

For the convenience of building before executing the tests, use the `test`/`test:node`/`test:browser` commands instead.

The live test pipeline or the unit test pipeline do not depend on the `test:` commands, `test:` commands are meant for dev convenience unlike `integration-test:` and `unit-test:` ones.